docs/source/tutorials/plots_intro.ipynb

```diff
@@ -530,6 +530,7 @@
    "id": "5211f6de-d447-46a8-8652-3c9fd77d7915",
```

Comment:
<blockquote>Combining aesthetics applied to variables and to dimensions we can generate a plot to compare the global parameters to the group parameters when we have a hierarchical model as in general the global parameter will be a scalar and the group ones a multidimensional one</blockquote><p>Looks like a very long sentence :D</p><p> </p><p>Do you want to have a comma before the "How close"...?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/arviz-devs/arviz-plots/pull/261/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>